cinema/views.py

```diff
@@ -173,3 +175,33 @@ def get_serializer_class(self):
 
     def perform_create(self, serializer):
         serializer.save(user=self.request.user)
+
+
+class MovieImageUploadView(views.APIView):
```

Comment:
It is possible but actually you are expected to create an @action on MovieViewSet - this approach is more simple and elegant